dbt/models/reporting/schema.yml

```diff
@@ -226,6 +226,13 @@ models:
           config:
             where: CAST(year AS int) < {{ var('test_qc_year_start') }}
             error_if: ">21" # as of 2024-04-01
+      - not_null:
+          name: reporting_vw_pin_value_long_tot_mv_not_null
+          column_name: tot_mv
+          config:
+            where: |
+              CAST(year AS int) < {{ var('test_qc_year_start') }} AND
+              year >= '2021'
```

Comment:
**question (non-blocking):** We have the test for things >= 2021 but the values constructed for >= 2020, why the difference?

Reply:
don't we only have board values in 2020?

I can update this to address that.